docs/servicenow.md

```diff
@@ -0,0 +1,149 @@
+# ticketutil.servicenow
+
+This document contains information on the methods available when working
+with a ServiceNow object. A list of the ServiceNow fields that have been
+tested when creating and editing tickets will be included. Because each
+instance of ServiceNow can have custom fields and custom values, some
+of the tested fields may not be applicable to certain instances of ServiceNow.
+Additionally, your ServiceNow instance may contain ticket fields that we have
+not tested. Keep in your mind different ServiceNow tables contain different
+fields with different properties. Mandatory field of one table is optional
```

Comment:
Change last sentence to "Mandatory fields in one table may be optional or non-existent in another table."